dspot/src/main/java/eu/stamp_project/utils/options/Configuration.java

```diff
@@ -23,6 +25,7 @@
 public class Configuration {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Configuration.class);
+    private static final DspotInformationCollector mongodbManager = new MongodbManager();
```

Comment:
by default it should be a NullCollector (an empty implementation that does nothing)